fansi/shared/src/main/scala/fansi/Fansi.scala

```diff
@@ -742,3 +761,35 @@ private[this] final class Trie[T](strings: Seq[(String, T)]){
   }
 }
 
+
+
+trait TrueColor extends Category {
+  /**
+    * 38 : front color
+    * 48 : background color
+    */
+  val zindex : Int
```

Comment:
Can we give this a bit of a longer, more appropriate name? Something like `trueColorCode` or something. `zindex` has nothing to do with what it does; it is neither an index, nor does it have anything to do with "z"